.rubocop.yml

```diff
@@ -217,6 +217,7 @@ Layout/SpaceAroundOperators:
     - '**/spec/actions_specs/xcodebuild_spec.rb'
 
 AllCops:
+  TargetRubyVersion: 2.4
```

Comment:
@giginet 

Thanks. You're right 👍 
Because default ruby version is 2.4.x in my development env, the PR instruction cause autocorrection by rubocop with ruby 2.4.x.

> bundle exec rubocop -a to ensure the code style is valid

Should we define `TargetRubyVersion: 2.0` in .rubocop.yml?